### ISSUE TITLE
When will the codes be available?

### ISSUE THREAD

**Issue by ilovecv** [yz93/LAVT-RIS#1]
Hi,

Exciting work! May I know when the codes will be released? Thank you very much!

**Comment by xmc-andy** [comment]
> h

This article's PWAM module is similar to the VLT's Query Generation Module, Language Gate (LG) similar to the gating mechanism of TV-Net. And the article does not use mainstream settings just for higher scores. In fact, this paper integrates EFN & TV-NET & VLT.

**Comment by Beirut-andy** [comment]
The PWAM module of this article is similar to the Query Generation Module of VLT, and the Language Gate (LG) is similar to the gating mechanism of TV-Net, and uses non-mainstream settings for higher performance scores. I don't think it's exciting. You will find that this article integrates EFN & VLT & TV net。

**Comment by yz93** [comment]
@ilovecv Thank you! We plan to release it soon, but we do have some other commitments at the moment. We don't think it will be too long though. Thanks for your comment again!

**Comment by yz93** [comment]
@Beirut-andy Hello! We've actually compared with VLT and EFN, _in quite specific ways_, in both the related work and the experiments sections. As for "TVNet" which you mentioned, we cannot properly identify which work you are referring to exactly without your giving a citation.

I want to emphasize that we used a fairly mainstream setting and your phrasing "uses non-mainstream settings for higher performance scores" is not only grossly ambiguous and inaccurate, but worse than that, seems to imply that we would intentionally employ unjust and unfair means to artificially improve the performance of our model, which is simply untrue.
 
While we are sorry that you did not like the work/did not find it exciting (a right which you are entitled to and which we absolutely respect), we must say that we worked hard in this project, and we believe in the merits that it has. We have also summarized our contributions clearly in the paper.

Thanks again for your comment.
